src/QuadForm/Quad/GenusRep.jl

```diff
@@ -155,7 +155,7 @@ function spinor_genera_in_genus(L, mod_out)
   # The smaller the element, the better
   for d in diagonal(Gr)
     if (iszero(spinornorm) && !iszero(d)) || (!iszero(d) && abs(norm(d)) < abs(norm(spinornorm)))
-      spinornorm = d
+      add!(spinornorm, spinornorm, d)
```

Comment:
Why add? Wasn't there a `=` before?

Reply:
`spinornorm` is already initialized to be zero, I thought that doing so would lighten the allocation process (?)
 `add!` will just modify the content of `spinornorm`

Reply:
No. Nothing is gained from this. It is actually worse with the change. Before it was just an assignment, which is a noop (no operation; does not cost anything). Now there is an addition, which is not for free.